app/views/keywords/_search.html.slim

```diff
@@ -0,0 +1,4 @@
+= form_tag(keywords_path, method: :get, id: 'searchForm', class: 'border-0 p-0 mt-5')
```

Comment:
how about `_search_form.`

Reply:
Could you please explain more about this?

Reply:
Fixed in a4b3460bd53cade6b560bc4d6f647ab8e2204e19